tendrl/commons/objects/cluster/atoms/delete_cluster_details/__init__.py

```diff
@@ -83,6 +84,31 @@ def run(self):
             # No cluster alerts, continue
             pass
 
+        # Remove cluster indexes for down node
+        try:
+            index_key = "/indexes/tags/tendrl/integration/%s" % integration_id
+            _node_ids = etcd_utils.read(
+                index_key
+            ).value
+            _node_ids = json.loads(_node_ids)
+            _node_arr = []
+            for _node_id in _node_ids:
+                node_obj = NS.tendrl.objects.NodeContext(
+                    node_id=_node_id
+                ).load()
+                if node_obj.status.lower() == "up":
+                    _node_arr.append(_node_id)
+                    with open("/root/out", "a") as f:
+                        f.write("came")
+            etcd_utils.write(index_key, json.dumps(_node_arr))
+        except (
+            etcd.EtcdKeyNotFound,
+            ValueError,
+            TypeError,
+            AttributeError
+        ):
+            pass
```

Comment:
Add a comment why you need to pass in these exception cases

Reply:
ack